src/app/ubs/ubs-admin/components/ubs-admin-export-details/ubs-admin-export-details.component.ts

```diff
@@ -41,7 +41,7 @@ export class UbsAdminExportDetailsComponent implements OnInit, OnDestroy, AfterV
     const hasNotValidFields = (everyFieldFilled && !someFieldFilled) || (!everyFieldFilled && someFieldFilled);
 
     Object.keys(this.exportDetailsDto.controls).forEach((controlName) => {
-      if (hasNotValidFields || isFormRequired) {
+      if ((hasNotValidFields || isFormRequired) && this.orderStatus !== OrderStatus.FORMED) {
```

Comment:
Are you sure that we need this condition? We check it here `const isFormRequired = !this.orderService.isStatusInArray(this.orderStatus, this.statuses);`

Reply:
"this.orderStatus !== OrderStatus.FORMED" makes the controllers in the exportDetailsDto group optional to complete if the status is "FORMED".

Reply:
The form isn't considered necessary when the status is "Formed", due to the variable - `hasNotValidFields` (within the if-statement). The array of statuses, where this form is deemed not required, includes the status "Formed". See the following defined statuses: private statuses = [OrderStatus.BROUGHT_IT_HIMSELF, OrderStatus.CANCELED, OrderStatus.FORMED]; In this context, the form's validation requirements are bypassed.

Reply:
When the status is 'FORMED', it is necessary to allow filling in one field while leaving the other two unfilled. If at least one field is filled, hasNotValidFields || isFormRequired will return true, and validation becomes mandatory. 

I have checked it multiple times, and it seems like I have come to the correct conclusion.

Reply:
Can try replacing hasNotValidFields || isFormRequired with hasNotValidFields && isFormRequired